api/src/modules/h3-data/h3-data.repository.ts

```diff
@@ -83,19 +72,13 @@ export class H3DataRepository extends Repository<H3Data> {
     indicatorH3Data: H3Data,
     materialH3Data: H3Data,
     calculusFactor: number,
-  ): Promise<H3IndexValueData> {
-    /**
-     * @note Using query in order to parse the resultant array into a single hash-map
-     * so the frontend has a constant time complexity when manipulating this data
-     * also, wee need to attach + info about the map: measure unit in which the values should be shown, indicator data, etc..
-     * We could not do this using streams
-     */
+  ): Promise<H3IndexValueData[]> {
     const riskmap = await getManager()
       .createQueryBuilder()
-      .select('materialh3.h3index')
+      .select('materialh3.h3index', 'h')
```

Comment:
again...

![triggered](https://media4.giphy.com/media/pYI1hSqUdcBiw/giphy.gif?cid=ecf05e47e4ikdybun6w8d7jyje7sl8a7fkzc2uzbs3w6m3zn&rid=giphy.gif&ct=g)

if we are keeping the names short so we can shrink the FE response, please add a note saying so

Reply:
https://github.com/Vizzuality/landgriffon/pull/101/files#diff-46aa1ad8bbc9bdbf2afdf5de5ca7928aedf234a1881a252f83e6914c3d6b06dfR5

It was added actualy sir 💇